radixdlt-java/src/main/java/com/radixdlt/client/application/translate/StatelessActionToParticlesMapper.java

```diff
@@ -8,6 +8,7 @@
  * to construct an atom.
  */
 public interface StatelessActionToParticlesMapper {
+	Observable<Action> sideEffects(Action action);
```

Comment:
Lacking documentation

Reply:
Both current implementations seem to just return Observable.empty.. what is this for? Why is it needed now if unused (future-proofing I assume)? 
Also, could use a default interface method here like:
```suggestion
	default Observable<Action> sideEffects(Action action) {
	    return Observable.empty();
	}
```